cerberus/tests/test_normalization.py

```diff
@@ -170,6 +170,15 @@ def test_nullables_dont_fail_coerce():
     assert_normalized(document, document, schema)
 
 
+def test_nullables_fail_coerce_on_non_null_values(validator):
+    schema = {'foo': {'coerce': lambda x: 1 / 0, 'nullable': True, 'type': 'integer'}}
```

Comment:
a coercion function that explicitly raised an exception with a simple message like "expected to fail" would make the intent clearer.

Reply:
Done